HW4-KZaballa.py

```diff
@@ -22,8 +24,20 @@
 # regression-in-python-3d3f8f06bf8
 
 
-def boolean_processing(predictor, response):
+def boolean_processing(predict_df, response):
     # fill in with logistic regression
+    for idx, column in enumerate(predict_df.T):
+        if (np.dtype(predict_df[idx]) == str) or (np.dtype(predict_df[idx]) == bool):
+            print("This variable is categorical")
+        elif (np.dtype(predict_df[idx]) == int) or (np.dtype(predict_df[idx]) == float):
+            print("This variable is continuous")
+        else:
+            print("This variable is ", np.dtype(predict_df[idx]))
+    logreg = LogisticRegression()
+    rfe = RFE(logreg, predict_df.columns.size)
+    rfe = rfe.fit(predict_df, response.values.ravel())
+    print(rfe.support_)
+    print(rfe.ranking_)
     return print("boolean processing")
 
 
```

Comment:
I saw you were still missing some plots and even the following: 

    # run weighted portion here
    # run ranking algorithm here
    # Instantiate model with 1000 decision trees
    rf = RandomForestClassifier(n_estimators=df.columns.size)
    # Train the model on training data
    rf.fit(df)
    print("Mean Absolute Error:", round(np.mean, 2), "degrees.")

The training model should be in the class lecture slides for random forest